grr/server/grr_response_server/gui/ui/components/client_details/client_diff.ts

```diff
@@ -202,3 +202,59 @@ export function getClientVersions(clientSnapshots: Client[]):
 
   return clientChanges;
 }
+
+function getFirstStringsJoinedPath(path: any[]): string {
```

Comment:
Don't use `any`, specify more precise types.

Reply:
`any` is the type of the provided argument, always. This is coming from the `deep-diff` implementation. Is there a workaround for this type of situation?

Reply:
Fair. Please add a comment on the preceeding line:
```
// tslint:disable-next-line:no-any Provided by deep-diff typing.
```